core/src/main/java/org/apache/iceberg/actions/SizeBasedDataRewriter.java

```diff
@@ -67,6 +68,8 @@ public Set<String> validOptions() {
   public void init(Map<String, String> options) {
     super.init(options);
     this.deleteFileThreshold = deleteFileThreshold(options);
+    this.deleteRatioThreshold = PropertyUtil.propertyAsDouble(
```

Comment:
we also need to validate that `0.0 > threshold <= 1.0`. That would also require some additional unit tests that check for valid/invalid parameters

Reply:
also `this.deleteRatioThreshold` needs to be used in `tooHighDeleteRatio()`